.ci/deps.sh

```diff
@@ -99,3 +99,6 @@ fi
 # Tailor (Swift) commands
 curl -fsSL https://tailor.sh/install.sh | sed 's/read -r CONTINUE < \/dev\/tty/CONTINUE=y/' > install.sh
 sudo bash install.sh
+
+# FlawFinder will be confused by our python 3 venv
+sudo sed -i 's/env python/python2/' /usr/bin/flawfinder
```

Comment:
This looks pretty specific.

IMHO `make`-ing things could lead to increased CI times.